docs/operations-guide/environment-variables.md

```diff
@@ -1,61 +1,810 @@
+
+#### `MB_API_KEY`
+
+Type: string<br>
+Default: `null`
+
+Middleware that enforces validation of the client via the request header `X-Metabase-Apikey`. If the header is available, then it’s validated against `MB_API_KEY`. When it matches, the request continues otherwise it’s blocked with a 403 Forbidden response.
+
+#### `MB_APPLICATION_COLORS`
+
+Only available in Enterprise Edition<br>
+Type: string<br>
+Default: `"{}"`
+
+JSON object of primary colors used in charts and throughout Metabase.
+
+#### `MB_APPLICATION_DB_MAX_CONNECTION_POOL_SIZE`
+
+Type: integer<br>
+Default: `15`<br>
+Since: 0.35.0
+
+Maximum number of connections to the Metabase application database.
+
+Change this to a higher value, if you notice that regular usage consumes close to or all connections. When all connections are in use, then Metabase might feel slow or unresponsive, when clicking around in the interface.
+
+Check the Metabase log, look for lines that contains the following `… App DB connections: 12/15 …`, where that example would indicate you’re using 12 out of 15 available connections.
+
+See [MB_JDBC_DATA_WAREHOUSE_MAX_CONNECTION_POOL_SIZE](#MB_JDBC_DATA_WAREHOUSE_MAX_CONNECTION_POOL_SIZE) for setting maximum connections to the databases setup in Metabase.
+
+#### `MB_APPLICATION_FAVICON_URL`
+
+Only available in Enterprise Edition<br>
+Type: string<br>
+Default: `"frontend_client/favicon.ico"`
+
+Path or URL to favicon file.
+
+#### `MB_APPLICATION_LOGO_URL`
+
+Only available in Enterprise Edition<br>
+Type: string<br>
+Default: `"app/assets/img/logo.svg"`
+
+Path or URL to logo file. For best results use SVG format.
+
+#### `MB_APPLICATION_NAME`
+
+Only available in Enterprise Edition<br>
+Type: string<br>
+Default: `"Metabase"`
+
+Replace the word `Metabase` wherever it appears with the defined string.
+
+#### `MB_ASYNC_QUERY_THREAD_POOL_SIZE`
+
+Type: integer<br>
+Default: `50`<br>
+Since: 0.35.0
+
+Maximum number of async Jetty threads. If not set, then [MB_JETTY_MAXTHREADS](#MB_JETTY_MAXTHREADS) will be used, otherwise it will use the default.
+
+#### `MB_BREAKOUT_BIN_WIDTH`
+
+Type: double<br>
+Default: `10.0`
+
+When using the default binning strategy for a field of type Coordinate (such as Latitude and Longitude), this number will be used as the default bin width (in degrees).
+
+#### `MB_BREAKOUT_BINS_NUM`
+
+Type: integer<br>
+Default: `8`
+
+When using the default binning strategy and a number of bins is not provided, this number will be used as the default.
+
+#### `MB_CHECK_FOR_UPDATES`
+
+Type: boolean<br>
+Default: `true`
+
+Identify when new versions of Metabase are available.
+
+#### `MB_CLIENT`
+
+Type: string<br>
+Default: `null`
+
+Used to recognize the Mac App client, which then defaults to `"OSX"`.
+
+#### `MB_COLORIZE_LOGS`
+
+Type: boolean<br>
+Default: `true`
+
+Color log lines. When set to `false` it will disable log line colors. This is disabled on Windows. Related to [MB_EMOJI_IN_LOGS](#MB_EMOJI_IN_LOGS)
```

Comment:
```suggestion
Color log lines. When set to `false` it will disable log line colors. This is disabled on Windows. Related to [MB_EMOJI_IN_LOGS](#MB_EMOJI_IN_LOGS).
```